components/ApplicationTable.tsx

```diff
@@ -1,14 +1,19 @@
 'use client'
 
-import { Applicant, Application, NextAction } from '@prisma/client'
-import { createColumnHelper } from '@tanstack/react-table'
-import React, { FC } from 'react'
+import { Applicant, Application, NextAction, User } from '@prisma/client'
```

Comment:
Given we are using just the types, we can narrow down the import:

```suggestion
import type { Applicant, Application, NextAction, User } from '@prisma/client'
```

Reply:
added in 3993a3331e005e1ae1fa3b79a364ca8ee3c2f24f